lib/electron-browser.js

```diff
@@ -14,18 +14,13 @@ function Electron (opt) {
   }
 
   AbstractBrowser.call(this)
-
-  var self = this
-  self._opt = opt
-  self.stats = {
-    passed: 0,
-    failed: 0
-  }
+  // TODO refactor this into AbstractBrowser?
```

Comment:
👍 Before that, let's refactor the options (`opt` and `conf`) of `SauceBrowser`, so that it only has one options object like the others.